docs/openpnp/calibration/10-vacuum-sensor/index.md

```diff
@@ -0,0 +1,94 @@
+
+# Vacuum Part Detection
+
+The bottom camera can do a great job of detecting if a part has been picked successfully. The LumenPnP comes with integrated vacuum sensors that can add an extra layer of verification to pick success. When the LumenPnP picks a part, if it detects a vacuum in the pneumatic line, it knows a component is making a seal with the nozzle tip and we've picked a part. 
+We switched vacuum sensors between v2 and v3 versions of the machine. Make sure you're using the correct config for your hardware.
```

Comment:
I'd move this under the Actuator setup heading.

```suggestion
Note that LumenPnP v2 has different vacuum sensors than v3, so the setup is a little different. Make sure to follow the steps for your machine version below.
```

Reply:
I agree, and I'll move it to the actuator setup heading